perfkitbenchmarker/packages/fio.py

```diff
@@ -153,4 +153,23 @@ def ParseResults(job_file, fio_json_result):
             sample.Sample('%s:latency' % metric_name,
                           job[mode]['lat']['mean'],
```

Comment:
In the parsing it transferred to json format, I believe has consistent unit (always use usec, even the number is large), I will double check